src/components/Personalization/in-app-message-actions/utils.js

```diff
@@ -9,7 +9,7 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-import { isNonEmptyArray, startsWith } from "../../../utils";
+import { isNonEmptyArray, queryString, startsWith } from "../../../utils";
```

Comment:
This will conflict with #1070 where both you and @shandilya3 made similar changes for the same thing

Reply:
I will close https://github.com/adobe/alloy/pull/1070